parachain/pallets/ethereum-beacon-client/src/mock.rs

```diff
@@ -1,22 +1,127 @@
+use hex_literal::hex;
+use std::{fs::File, path::PathBuf};
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0
// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
use hex_literal::hex;
use std::{fs::File, path::PathBuf};
```

License should be the first lines in the file.